skyuxconfig-schema.json

```diff
@@ -279,6 +279,10 @@
       "description": "Specifies a path to reference SKY UX components in Angular. This property is specific to the SKY UX docs site and is for internal Blackbaud use only.",
       "type": "string"
     },
+    "incompatibleDependencies": {
```

Comment:
I'm not sure I love the name `incompatibleDependencies`, since there could be multiple interpretations of what that could mean. What about adding a property to the existing `librarySettings` property named something like `needsTranspilation`?

Reply:
Decided to go with `dependenciesForTranspilation`.